h/templates/client/annotation.html

```diff
@@ -26,17 +26,17 @@
            target="_blank" ng-if="vm.group() && vm.group().url" href="{{vm.group().url}}">
           <i class="h-icon-group"></i><span class="annotation-header__group-name">{{vm.group().name}}</span>
         </a>
-        <span ng-show="vm.isPrivate()"
+        <span ng-show="vm.isPrivate"
               title="This annotation is visible only to you.">
           <i class="h-icon-lock"></i><span class="annotation-header__group-name" ng-show="!vm.group().url">Only me</span>
         </span>
         <i class="h-icon-border-color" ng-show="vm.isHighlight() && !vm.editing()" title="This is a highlight. Click 'edit' to add a note or tag."></i>
         <span class="annotation-citation"
-              ng-bind-html="vm.document | documentTitle"
+              ng-bind-html="vm.document() | documentTitle"
```

Comment:
The use of a filter here is unnecessary, this can just be `vm.document().title`, `vm.document().domain`. I think that would enable removing the `documentTitle` and `documentDomain` filters from the app entirely.